rollup/core/classes/RollupAsyncProcessor.cls

```diff
@@ -584,14 +593,71 @@ global virtual without sharing class RollupAsyncProcessor extends Rollup impleme
     this.processDeferredRollups();
   }
 
+  protected virtual Map<String, CalcItemBag> getCalcItemsByLookupField(RollupAsyncProcessor roll, Set<String> uniqueLookupObjectFields) {
```

Comment:
no changes to this method, I just moved it up because it's now protected instead of private